app/src/main/java/app/grapheneos/camera/capturer/ImageSaver.kt

```diff
@@ -123,7 +129,8 @@ class ImageSaver(
             handleError(e)
             return
         }
-
+        Log.i("TAG", "Done with image $photosInQueue")
+        --photosInQueue
```

Comment:
It's incorrect to modify a regular variable from multiple threads without synchronization.
Line 78 is executed on the main thread, this line -- on a background thread.

Reply:
I'll soon replace the regular `Int` variable with `AtomicInteger` and its other usages to ensure thread safety.